sdk/eventgrid/azure-eventgrid/azure/eventgrid/models/__init__.py

```diff
@@ -0,0 +1,300 @@
+    'MediaJobOutputCanceledEventData',
+    'MediaJobOutputCancelingEventData',
+    'MediaJobOutputErroredEventData',
+    'MediaJobOutputFinishedEventData',
+    'MediaJobOutputProcessingEventData',
+    'MediaJobOutputProgressEventData',
+    'MediaJobOutputScheduledEventData',
+    'MediaJobOutputStateChangeEventData',
+    'MediaJobProcessingEventData',
+    'MediaJobScheduledEventData',
+    'MediaJobStateChangeEventData',
+    'MediaLiveEventConnectionRejectedEventData',
+    'MediaLiveEventEncoderConnectedEventData',
+    'MediaLiveEventEncoderDisconnectedEventData',
+    'MediaLiveEventIncomingDataChunkDroppedEventData',
+    'MediaLiveEventIncomingStreamReceivedEventData',
+    'MediaLiveEventIncomingStreamsOutOfSyncEventData',
+    'MediaLiveEventIncomingVideoStreamsOutOfSyncEventData',
+    'MediaLiveEventIngestHeartbeatEventData',
+    'MediaLiveEventTrackDiscontinuityDetectedEventData',
+    'RedisExportRDBCompletedEventData',
+    'RedisImportRDBCompletedEventData',
+    'RedisPatchingCompletedEventData',
+    'RedisScalingCompletedEventData',
+    'ResourceActionCancelData',
+    'ResourceActionFailureData',
+    'ResourceActionSuccessData',
+    'ResourceDeleteCancelData',
+    'ResourceDeleteFailureData',
+    'ResourceDeleteSuccessData',
+    'ResourceWriteCancelData',
+    'ResourceWriteFailureData',
+    'ResourceWriteSuccessData',
+    'SMSDeliveryAttemptProperties',
+    'SMSDeliveryReportReceivedEventData',
+    'SMSEventBaseProperties',
+    'SMSReceivedEventData',
+    'ServiceBusActiveMessagesAvailableWithNoListenersEventData',
+    'ServiceBusDeadletterMessagesAvailableWithNoListenersEventData',
+    'SignalRServiceClientConnectionConnectedEventData',
+    'SignalRServiceClientConnectionDisconnectedEventData',
+    'StorageBlobCreatedEventData',
+    'StorageBlobDeletedEventData',
+    'StorageBlobRenamedEventData',
+    'StorageDirectoryCreatedEventData',
+    'StorageDirectoryDeletedEventData',
+    'StorageDirectoryRenamedEventData',
+    'StorageLifecyclePolicyActionSummaryDetail',
+    'StorageLifecyclePolicyCompletedEventData',
+    'SubscriptionDeletedEventData',
+    'SubscriptionValidationEventData',
+    'SubscriptionValidationResponse',
+    'WebAppServicePlanUpdatedEventData',
+    'WebAppServicePlanUpdatedEventDataSku',
+    'WebAppUpdatedEventData',
+    'WebBackupOperationCompletedEventData',
+    'WebBackupOperationFailedEventData',
+    'WebBackupOperationStartedEventData',
+    'WebRestoreOperationCompletedEventData',
+    'WebRestoreOperationFailedEventData',
+    'WebRestoreOperationStartedEventData',
+    'WebSlotSwapCompletedEventData',
+    'WebSlotSwapFailedEventData',
+    'WebSlotSwapStartedEventData',
+    'WebSlotSwapWithPreviewCancelledEventData',
+    'WebSlotSwapWithPreviewStartedEventData',
+    'AppAction',
+    'AppServicePlanAction',
+    'AsyncStatus',
+    'MediaJobErrorCategory',
+    'MediaJobErrorCode',
+    'MediaJobRetry',
+    'MediaJobState',
+    'StampKind',
```

Comment:
Debating on whether we want this to be manual or not.  It seems like we could use some semblence of that importlib stanza I showed you?  Granted it seems... wrong to be doing automatic population of an import but I'm also not settled on the super-long-list of types.  I'd be curious to get @annatisch 's take on this as far as "python best practices" goes.

Reply:
you know my hatred towards long enumerations, but init makes exception IMO - would like to have more control over what we are exposing

Reply:
I'm fine to play this by ear.  My gut says in a few <months, years> as new services trickle in we'll end up wanting a less manual update process but that can very much be down the road.